src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  width: $cardSize;
+  height: $cardHeight;
+  display: flex;
+  margin-right: $gap;
+  margin-bottom: 48px;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.